src/semantic-router/pkg/utils/classification/classifier_test.go

```diff
@@ -406,3 +406,49 @@ var _ = Describe("PIIClassification", func() {
 		})
 	})
 })
+
+func TestUpdateBestModel(t *testing.T) {
+
+	classifier := &Classifier{}
+
+	bestScore := 0.5
+	bestQuality := 0.5
+	bestModel := "old-model"
+
+	classifier.updateBestModel(0.8, 0.9, "new-model", &bestScore, &bestQuality, &bestModel)
+	if bestScore != 0.8 || bestQuality != 0.9 || bestModel != "new-model" {
+		t.Errorf("update: got bestScore=%v, bestQuality=%v, bestModel=%v", bestScore, bestQuality, bestModel)
+	}
+
+	classifier.updateBestModel(0.7, 0.7, "another-model", &bestScore, &bestQuality, &bestModel)
+	if bestScore != 0.8 || bestQuality != 0.9 || bestModel != "new-model" {
+		t.Errorf("not update: got bestScore=%v, bestQuality=%v, bestModel=%v", bestScore, bestQuality, bestModel)
+	}
+}
+
+func TestForEachModelScore(t *testing.T) {
+
+	c := &Classifier{}
+	cat := &config.Category{
+		ModelScores: []config.ModelScore{
```

Comment:
the order is exactly the same as you check in line 445, this makes the test untrustable, can you make it unordered?

Reply:
sorry, i don`t undersatnd.

i used the append func here to append models, and the appended content should be the grid.

Reply:
Aha, I got u sorry I think misunderstood it, plz go ahead